src/App.tsx

```diff
@@ -1,32 +1,46 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
+import { useEffect, useState } from 'react'
 import './App.css'
+import { CartPage } from './domain/cart/CartPage'
+import { BooksPage } from './domain/books/BooksPage'
+import Navbar from './components/navbar/navbar'
+import { getBooks, useAppDispatch, useAppSelector } from './domain/books/booksSlice'
+import { Book } from './domain/books/types'
+import { addToCart,removeFromCart} from './domain/cart/cartSlice'
+import { CartItem, CartState } from './domain/cart/types'
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [route, setRoute] = useState<number>(0)
```

Comment:
Declare the routes in src/router/Router.tsx and use react-router-dom to handle them